metaflow/plugins/argo/argo_live_run_class.py

```diff
@@ -40,8 +41,8 @@ def __init__(
             alt_flow_id_info = {}
 
         template_name = None
-        if 'template_name' in alt_flow_id_info:
-            template_name = alt_flow_id_info['template_name']
+        if "template_name" in alt_flow_id_info:
+            template_name = alt_flow_id_info["template_name"]
```

Comment:
I think `alt_flow_id_info` can be `template_name` directly.

Reply:
done